includes/hide.php

```diff
@@ -204,13 +204,13 @@ function fastwc_should_hide_pdp_button_for_unsupported_product( $should_hide, $p
 function fastwc_should_hide_pdp_button_for_out_of_stock_product( $should_hide, $product ) {
 
 	if ( ! $should_hide ) {
-		$stock_status = method_exists( $product, 'get_stock_status' ) ? $product->get_stock_status : '';
+		$stock_status = method_exists( $product, 'get_stock_status' ) ? $product->get_stock_status() : '';
 
```

Comment:
We need to define `$product_id` here just in case `$product` does not have the `get_id` method.

```suggestion
		$product_id = method_exists( $product, 'get_id' ) ? $product->get_id() : 0;

```